### ISSUE TITLE
Monster memory display glitch

### ISSUE THREAD

**Issue by angband-import** [angband/angband#781]
**Reported by anonymous on 12 Jul 2007 06:24 UTC**
When recalling memory of a monster, the leftmost character of the description is used for indentation.  However, that character is not written; it's just skipped.  This means that the underlying characters (from e.g. the player status on the left of the main map view) blurs into the monster description text, like this:

```
The White worm mass ('w')/('w'): [(r)ecall, ESC]
You have killed at least 18 of these creatures.
MIt is a large slimy mass of worms.
NThis natural creature is normally found on dungeon level 1, and moves extremely erratically, and
Lslowly.  A kill of this creature is worth 0.40 points for a 5th level character.  It breeds
Nexplosively.  It can crawl on you to poison.
```

**Comment by angband-import** [comment]
**Comment by ctate\@acm.org on 15 Jul 2007 04:54 UTC**
For recordkeeping, I'm the one who reported this.  I forgot to change the 'submitter' field; sorry.

**Comment by angband-import** [comment]
**Modified by takkaria on 15 Jul 2007 22:05 UTC**

**Comment by angband-import** [comment]
**Modified by ajps on 18 Jul 2007 21:47 UTC**

**Comment by angband-import** [comment]
**Comment by ajps on 18 Jul 2007 21:55 UTC**
Fixed in [351] merged in [5dfe994] (SVN r399).
